android/src/main/java/io/sentry/react/RNSentryModuleImpl.java

```diff
@@ -616,6 +621,48 @@ public void disableNativeFramesTracking() {
         }
     }
 
+    public WritableMap startProfiling() {
+        final WritableMap result = new WritableNativeMap();
+        try {
+            HermesSamplingProfiler.enable();
+            result.putBoolean("started", true);
+        } catch (Throwable e) {
+            result.putBoolean("started", false);
+            result.putString("error", e.toString());
+        }
+        return result;
+    }
+
+    public WritableMap stopProfiling() {
+        final WritableMap result = new WritableNativeMap();
+        try {
+            HermesSamplingProfiler.disable();
+
+            final File output = File.createTempFile(
+                "sampling-profiler-trace", ".cpuprofile", reactApplicationContext.getCacheDir());
```

Comment:
If we always use the same file, are we overwriting and/or deleting the old files? otherwise, it might be better to use a generated name to avoid overwriting/using the wrong one.

Reply:
We are overwriting the file, but since there is only one sampling profiler and always when we stop we read the data and send them to JS, it should not be a problem.

Reply:
Can we delete the file in the finally block at least?